demo/ContainerInteractor.swift

```diff
@@ -122,21 +132,14 @@ extension ContainerInteractor: ContainerDelegate {
     func backButtonPressed() {
         presenter.dismissCurrent()
         presenter.presentBackButton(isVisible: false)
```

Comment:
Seems like we're configuring back button before every call of `present(screen:)`. Should we move the configuration inside of the method?